distribution/kernel/src/assembly/bin.xml

```diff
@@ -158,6 +158,7 @@
                 <include>javax.activation:activation:jar</include>
                 <include>org.apache.geronimo.specs:geronimo-jta_1.1_spec:jar</include>
                 <include>javax.annotation:javax.annotation-api:jar</include>
+                <include>org.wso2.orbit.sun.xml.ws:jaxws-ri:jar</include>
```

Comment:
@afkham @jsdjayanga 
Can you please evaluate this change.
We need to expose the xml ws lib non-osgi way using java modules import mechanism for newer version of libraries, 
e.g. Open SAML

Reply:
With this fix, mentioned packages will be exposed to OSGi environment via the system-bundle with the version 0.0.0. This is ok. We have done the same for several other jars in our Java 11 migration.

However, you might have to check the dependencies of jaxws-ri. If there are any dependencies, you might have to do the same for those dependencies as well.